src/mono/mono/mini/mini-llvm.c

```diff
@@ -8704,12 +8710,6 @@ MONO_RESTORE_WARNING
 			values [ins->dreg] = call_intrins (ctx, id, args, "");
 			break;
 		}
-		case OP_XOP_OVR_X_X: {
-			IntrinsicId iid = (IntrinsicId) ins->inst_c0;
```

Comment:
These probably can be moved outside the ifdefs.

Reply:
Where were they before?